### DISCUSSION TITLE
What is difference between zellij and tmux with similar configuration?

### DISCUSSION THREAD

**Discussion by santosh** [zellij-org/zellij#1701] [Q&A]
I have just tried zellij. What I feel at first use it, it is same as tmux which I am using in day to day work. What exactly is the difference?

**Comment by imsnif** [comment]
EDIT 2024: I have now elaborated my views on this matter in [the following blogpost](https://poor.dev/blog/why-zellij/).

**Reply by Stef16Robbe** [reply]
It's exactly these differences that instantly made me fall in love with Zellij. Sane defaults, an actual user-friendly interface instead of "go fetch", and added features that makes it much easier to use.

**Reply by Lockszmith-GH** [reply]
@imsnif Would you please create a pinned post titled "Zellij is not a XXX replacement"?
I found myself directing people to this response multiple times, feels like it would be easier to just type `#` followed by a number in future responses.

**Reply by imsnif** [reply]
Hey @Lockszmith-GH - your comment motivated me to write a blog post about it. I edited the above comment accordingly. I hope it will answer everyone's concerns.

On a different subject - thank you for being so helpful in the discussions and issues. If you would like to email me your posting address to aram@poor.dev, I would be happy to send you some Zellij stickers.

**Reply by Lockszmith-GH** [reply]
Thanks @imsnif, I sent you an email. I feel uncomfortable seeing your email address here, so please edit it out :)

Also, there is a typo in the blog (great blog post BTW): 'preditable' probably should be 'predictable'.

**Comment by alexmozaidze** [comment]
More is good

**Comment by ixzh** [comment]
my setup on android is 3 pane on left right and bottom. In tmux with mouse enabled, the bottom pane keeps on being squeezed after each app switch. with Zellij all stay put: it gives me inner peace.

**Comment by severgun** [comment]
Main difference is that it is not available on remote server.
